src/cmd/internal.go

```diff
@@ -113,6 +114,16 @@ var uiCmd = &cobra.Command{
 	},
 }
 
+var isValidHostname = &cobra.Command{
+	Use:   "is-valid-hostname",
+	Short: "Checks if the machine hostname is RFC1123 compliant",
+	Run: func(cmd *cobra.Command, args []string) {
+		if valid := utils.IsValidHostName(); !valid {
+			message.Fatal(nil, "The hostname is not RFC1123 compliant")
```

Comment:
Per the above (and this is what the user is more likely to see anyway).

```suggestion
			hostname, _ := os.Hostname()
			message.Fatalf(nil, "The hostname '%s' is not valid. Ensure the hostname meets RFC1123 requirements https://www.rfc-editor.org/rfc/rfc1123.html.", hostname)
```